server/chat/background/suggestion_extractor.py

```
@@ -269,11 +273,15 @@ def _parse_suggestions_response(self, content: Any) -> List[Suggestion]:
-            if command and not is_command_safe(command):
-                logger.warning(
-                    f"[SuggestionExtractor] Flagged dangerous command as high risk: {command[:100]}"
-                )
-                risk = "high"
+            if command:
+                if re.search(r'<[a-zA-Z][a-zA-Z0-9_-]*>', command):
+                    logger.info("[SuggestionExtractor] Stripped command with placeholders: %s", command[:100])
+                    command = None
+                elif not is_command_safe(command):
+                    logger.warning(
+                        f"[SuggestionExtractor] Flagged dangerous command as high risk: {command[:100]}"
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)